doc/md/schema-def.md

```diff
@@ -58,6 +58,11 @@ the root directory of your project, and can be generated by `entc` as follows:
 go run entgo.io/ent/cmd/ent init User Group
 ```
 
+:::note 
+Pleae note, that `Client` is a reserved Go struct and cannot be used as schema name. For database tables
+named `client`, please use an annotation as mentioned [here](schema-annotations.md#custom-table-name).
+:::
```

Comment:
What's so special about `Client`? Maybe expand the godoc of [gen.ValidSchemaName](https://pkg.go.dev/entgo.io/ent/entc/gen#ValidSchemaName) and link to it?